content/en/docs/tasks/debug-application-cluster/monitor-node-health.md

```diff
@@ -1,133 +1,117 @@
 ---
+title: Monitor Node Health
+content_type: task
 reviewers:
 - Random-Liu
 - dchen1107
-content_type: task
-title: Monitor Node Health
 ---
 
 <!-- overview -->
 
-*Node problem detector* is a [DaemonSet](/docs/concepts/workloads/controllers/daemonset/) monitoring the
-node health. It collects node problems from various daemons and reports them
-to the apiserver as [NodeCondition](/docs/concepts/architecture/nodes/#condition)
+*Node problem detector* is a daemon for monitoring and reporting about a node's health.
+You can run node problem detector as a `DaemonSet`
+or as a standalone daemon. Node problem detector collects information about node problems from various daemons
+and reports these conditions to the API server as [NodeCondition](/docs/concepts/architecture/nodes/#condition)
 and [Event](/docs/reference/generated/kubernetes-api/{{< param "version" >}}/#event-v1-core).
 
-It supports some known kernel issue detection now, and will detect more and
-more node problems over time.
-
-Currently Kubernetes won't take any action on the node conditions and events
-generated by node problem detector. In the future, a remedy system could be
-introduced to deal with node problems.
-
-See more information
-[here](https://github.com/kubernetes/node-problem-detector).
-
-
+To learn how to install and use the node problem detector, see the
+[Node problem detector project documentation](https://github.com/kubernetes/node-problem-detector).
 
 ## {{% heading "prerequisites" %}}
 
-
-{{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
-
-
+{{< include "task-tutorial-prereqs.md" >}}
 
 <!-- steps -->
 
 ## Limitations
 
-* The kernel issue detection of node problem detector only supports file based
-kernel log now. It doesn't support log tools like journald.
+* Node problem detector only supports file based kernel log.
```

Comment:
nit: Here and everything title case "Node Problem Detector"

Reply:
OK. I can create a follow-up PR. Thanks for reviewing!